packages/patternfly-4/react-core/src/demos/FilterTableDemo.md

```diff
@@ -0,0 +1,388 @@
+      ]
+    };
+
+    this.onDelete = (type = '', id = '') => {
+      if (type) {
+        this.setState(prevState => {
+          prevState.filters[type.toLowerCase()] = prevState.filters[type.toLowerCase()].filter(s => s !== id);
+          return {
+            filters: prevState.filters
+          };
+        });
+      } else {
+        this.setState({
+          filters: {
+            location: [],
+            name: [],
+            status: []
+          }
+        });
+      }
+    };
+
+    this.onCategoryToggle = isOpen => {
+      this.setState({
+        isCategoryDropdownOpen: isOpen
+      });
+    };
+
+    this.onCategorySelect = event => {
+      this.setState({
+        currentCategory: event.target.innerText,
+        isCategoryDropdownOpen: !this.state.isCategoryDropdownOpen
+      });
+    };
+
+    this.onFilterToggle = isOpen => {
+      this.setState({
+        isFilterDropdownOpen: isOpen
+      });
+    };
+
+    this.onFilterSelect = event => {
+      this.setState({
+        isFilterDropdownOpen: !this.state.isFilterDropdownOpen
+      });
+    };
+
+    this.onInputChange = newValue => {
+      this.setState({ inputValue: newValue });
+    };
+
+    this.onRowSelect = (event, isSelected, rowId) => {
+      let rows;
+      if (rowId === -1) {
+        rows = this.state.rows.map(oneRow => {
+          oneRow.selected = isSelected;
+          return oneRow;
+        });
+      } else {
+        rows = [...this.state.rows];
+        rows[rowId].selected = isSelected;
+      }
+      this.setState({
+        rows
+      });
+    };
+
+    this.onStatusSelect = (event, selection) => {
+      const checked = event.target.checked;
+      this.setState(prevState => {
+        const prevSelections = prevState.filters['status'];
+        return {
+          filters: {
+            ...prevState.filters,
+            ['status']: checked ? [...prevSelections, selection] : prevSelections.filter(value => value !== selection)
+          }
+        };
+      });
+    };
+
+    this.onNameInput = event => {
+      if (event.key && event.key !== 'Enter') {
+        return;
+      }
+
+      const { inputValue } = this.state;
+      this.setState(prevState => {
+        const prevFilters = prevState.filters['name'];
+        return {
+          filters: {
+            ...prevState.filters,
+            ['name']: prevFilters.includes(inputValue)
+              ? prevFilters.filter(value => value !== inputValue)
+              : [...prevFilters, inputValue]
+          },
+          inputValue: ''
+        };
+      });
+    };
+
+    this.onLocationSelect = (event, selection) => {
+      this.setState(prevState => {
+        return {
+          filters: {
+            ...prevState.filters,
+            ['location']: [selection]
+          }
+        };
+      });
+      this.onFilterSelect();
+    };
+  }
+
+  buildCategoryDropdown() {
+    const { isCategoryDropdownOpen, currentCategory } = this.state;
+
+    return (
+      <Dropdown
```

Comment:
This dropdown should also be in a DataToolbarItem or some other DataToolbar wrapper component

Reply:
It probably should be in the same DataToolbarGroup variant='filter-group' as the other dropdown.